lib/core/rollup.config.js

```diff
@@ -0,0 +1,298 @@
+import fs from "fs";
+import path from "path";
+import vue from "rollup-plugin-vue";
+import alias from "@rollup/plugin-alias";
+import commonjs from "@rollup/plugin-commonjs";
+import resolve from "@rollup/plugin-node-resolve";
+import replace from "@rollup/plugin-replace";
+import babel from "@rollup/plugin-babel";
+import PostCSS from "rollup-plugin-postcss";
+import simplevars from "postcss-simple-vars";
+import postcssImport from "postcss-import";
+import minimist from "minimist";
+import postcssUrl from "postcss-url";
+import url from "@rollup/plugin-url";
+import nested from "postcss-nested";
+import { terser } from "rollup-plugin-terser";
+import autoprefixer from "autoprefixer";
+import typescript from "rollup-plugin-typescript2";
+import css from "rollup-plugin-css-only";
+
+const postcssConfigList = [
+  postcssImport({
+    resolve(id, basedir) {
+      // resolve alias @css, @import '@css/style.css'
+      // because @css/ has 5 chars
+      if (id.startsWith("@css")) {
+        return path.resolve("./src/assets/styles/css", id.slice(5));
+      }
+
+      // resolve node_modules, @import '~normalize.css/normalize.css'
+      // similar to how css-loader's handling of node_modules
+      if (id.startsWith("~")) {
+        return path.resolve("./node_modules", id.slice(1));
+      }
+
+      // resolve relative path, @import './components/style.css'
+      return path.resolve(basedir, id);
+    }
+  }),
+  simplevars,
+  nested,
+  postcssUrl({ url: "inline" }),
+  autoprefixer({
+    overrideBrowserslist: "> 1%, IE 6, Explorer >= 10, Safari >= 7"
+  })
+];
+
+const argv = minimist(process.argv.slice(2));
+
+const projectRoot = path.resolve(__dirname, ".");
+
+let postVueConfig = [
+  // Process only `<style module>` blocks.
+  PostCSS({
+    modules: {
+      generateScopedName: '[local]___[hash:base64:5]',
+    },
+    include: /&module=.*\.css$/,
+  }),
+  // Process all `<style>` blocks except `<style module>`.
+  PostCSS({ include: /(?<!&module=.*)\.css$/,
+    plugins:[
+      ...postcssConfigList
+    ]
+  }),
+  url({
+    include: [
+      '**/*.svg',
+      '**/*.png',
+      '**/*.gif',
+      '**/*.jpg',
+      '**/*.jpeg'
+    ]
+  }),
```

Comment:
I don't think we should bring along bitmap images like PNG or JPG. Thing is we either:
- Leave them as is (binary files), which means the users may need to setup proper loaders, even for files inside node_modules. Some tools like Nextjs will complain about this
- Or we bundled them inside JS code, which means they will be inlined as base64 or something, which makes the sizing a little bit hard to control

Anyway, my point is, I don't know how to handle bitmap image files in a distributable libs. If you know what you're doing here, just go ahead